clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StoreResourceDefinition.java

```diff
@@ -137,6 +138,8 @@ public ModelNode transformOperation(ModelNode operation) {
                 }
             };
             builder.addRawOperationTransformationOverride(MapOperations.MAP_PUT_DEFINITION.getName(), new SimpleOperationTransformer(removePropertyTransformer));
+
+            builder.getAttributeBuilder().setDiscard(DiscardAttributeChecker.ALWAYS, PROPERTIES);
```

Comment:
Good catch, the operation transformer does not kick in. We will need an attribute converter to be kind here.